addons/nuxeo-platform-spreadsheet/nuxeo-platform-spreadsheet-web/src/main/js/app/app.js

```diff
@@ -94,23 +86,23 @@ function doQuery() {
   });
 }
 
-function run(baseURL = '/nuxeo', username = null, password = null) {
-
+function _run({ baseURL = '/nuxeo', username = null, password = null, resultLayout, resultColumns, pageProviderName, queryParameters, searchDocument, sortInfos }) {
   // Setup our connection
   var nx = new Connection(baseURL, username, password);
   nx.schemas(['*']);
 
+  isStandalone = !pageProviderName;
+
   setupUI();
 
+  console.log("Here")
```

Comment:
do we need this console.log?

Reply:
Yes. This `console.log()` is crucial 😛 . Removed anyway 😉